fudge-ratio/src/main/webapp/script.js

```diff
@@ -71,6 +81,57 @@ buttonClear.addEventListener("click", () => {
   cleanSessionState();
 });
 
+buttonAddHour.addEventListener("click", () => {
+    durationHour += 1;
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
+});
+
+buttonMinusHour.addEventListener("click", () => {
+    if(durationHour > 0) {
+        durationHour -= 1;
+    }
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
+});
+
+buttonAddMinute.addEventListener("click", () => {
+    if(durationMin == 59) {
+        durationMin = 0;
+        durationHour += 1;
+    } else {
+        durationMin += 1;
+    }
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
+});
+
+buttonMinusMinute.addEventListener("click", () => {
+    if(durationMin > 0) {
+        durationMin -= 1;
+    }
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
+});
+
+buttonAddSecond.addEventListener("click", () => {
+    if(durationSec == 59) {
```

Comment:
This seems like a repetitive kind of work ....([part 2])

Maybe when we have modules we could do a class time that is able to handle this in all places of the app